assets/js/vue/img.js

```diff
@@ -113,7 +115,7 @@ export default {
 
         this.dimensions = dimensions;
 
-        height && width ? this.buildSrcSet(preset, transformationsString) : null;
+        height && width && !this.isSvgCloudinary() ? this.buildSrcSet(preset, transformationsString) : null;
```

Comment:
Glaube an der Stelle ist es etwas zu wenig. Jetzt läuft der Code in getSetup, baut dort einen TransformationString zusammen und holt sich ein preset, welches Informationen sind, die gar nicht berechnet werden müssen für ein SVG. Ein SVG kann so gesehen auch kein Preset haben. Denke das muss man im onload getrennt betrachten.